### PR TITLE
Updated form instruction styles. Updated z-index bug

### LINKED ISSUE #1963: Form instructions - design tightening
Tweak instruction style to match comp:

- [x] Remove overall white bg and stroke
- [x] Put white bg on the instructions and the yes/no text
- [x] More margin above the yes/no text to push border up
- [x] Increase stroke weight around instructions to 2px

Dev:
screen shot 2017-09-21 at 10 22 20 am

Comp:
screen shot 2017-09-21 at 10 22 50 am

Desired in browser comp:
screen shot 2017-09-21 at 10 28 45 am

### PR DESCRIPTION
**Pull request by floresj**
Resolves #1963. Also fixes issue where elements were being rendered on top of the disclaimer. Refer to https://github.com/truetandem/e-QIP-prototype/pull/2005#issuecomment-333719285 for details.

### CODE REVIEW

**Comment by bmallred** [comment]
What does this have to do w/ _updated z-index bug_? The referenced issue had nothing to do with z-index and was merely style adjustments, right?

**Comment by floresj** [comment]
That was something I had noticed. When you reduced the width of the page with the form instructions, you sometimes could see some of the components. Here's an example:

selection_474

I forgot to include that in the description at the top, my bad.

File: src/components/Form/Introduction/Introduction.scss [hunk 1]
```diff
@@ -1,14 +1,22 @@
 @import 'eqip-colors';
 
+.introduction-modal {
+  z-index: 9999;
+}
+
 .introduction-modal .modal {
   background-color: rgba(255, 255, 255, 0.95) !important;
+  z-index: 9999;
```
Comment (bmallred) [inline src/components/Form/Introduction/Introduction.scss]: Is this redundant? See line 4

Comment (floresj) [reply src/components/Form/Introduction/Introduction.scss]: I thought I had removed that. I'm getting all tangled up with my branches. Will remove :+1:

Comment (floresj) [reply src/components/Form/Introduction/Introduction.scss]: Oh nvm. Was looking at something else. So there's a style for `.modal` that has `rgba(255, 255, 255, 0.9)`. This one appears to be overriding it with more opacity `0.95`. Should I just change the `.modal` to use `0.95` and remove the `background-color` for `.introduction-modal .modal`?

**Pull request merged**